uberfire-api/src/main/java/org/uberfire/debug/Debug.java

```diff
@@ -44,6 +44,7 @@
      */
     public static String toMemorableString(long num) {
         StringBuilder memorable = new StringBuilder();
+        num=Math.abs(num);
```

Comment:
formatting should be `num = Math.abs(num);`. Even more, from my point of view it would be better not to change value of the method input parameter and just rewrite the line in cycle `int chunk = (int) (Math.abs(num) & 0xf);`
WDYT?

Reply:
Thanks @jomarko , formatting change is done.

Since method uses input parameter as variable and we need to change at multiple lines.
or else we need to assign input parameter to a new variable and use it.
i have added new variable and updated the code. Plz review it